src/components/BuildConfigRevisionPages/BuildConfigRevisionPages.tsx

```diff
@@ -36,13 +38,23 @@ export const BuildConfigRevisionPages = ({ componentId = 'r1' }: IBuildConfigDet
     { componentId, mandatoryQueryParams: listMandatoryQueryParams.pagination }
   );
 
+  useEffect(() => {
+    const pageIndex = new URLSearchParams(search).get('r1-pageIndex');
+    if (serviceContainerBuildConfigRevisions.data?.content?.length && pageIndex === '1') {
+      const latestRev = serviceContainerBuildConfigRevisions.data.content[0]?.rev;
+      setIsCurrentRevision(String(latestRev) === revisionId);
+    } else {
```

Comment:
Is `else` needed?

Reply:
I think if the else branch is omitted, when the condition is not met, the state of isCurrentRevision will not be updated and may retain its previous value when the component was dynamically updated.